### PR TITLE
Updated sum.py in feature branch

### PR DESCRIPTION
**Pull request by lit-tsa**

### CODE REVIEW

File: dev/sum.py [hunk 1]
```diff
@@ -1 +1,3 @@
-
+a=10
+b=20
+print "The sum of a and b is {a+b}"
```
Comment (dav-bon) [inline dev/sum.py:3]: Dont think that print will work.

**Comment by dav-bon** [comment]
But lets just try.

**Review by dav-bon — APPROVED** [review]
Ok

File: dev/sum.py (hunk 1 above)
Comment (lit-tsa) [reply dev/sum.py:3]: It will work.

**Comment by lit-tsa** [comment]
Testing this comment

**Pull request merged**